docs/usage/fields.md

```diff
@@ -246,4 +274,99 @@ print(user)
 #> first_name='John' last_name='Doe'
 ```
 
+## Number Constraints
+
+There are fields that can be used to constrain numbers:
+
+* `gt` - greater than
+* `lt` - less than
+* `ge` - greater than or equal to
+* `le` - less than or equal to
+* `multiple_of` - a multiple of the given number
+* `allow_inf_nan` - allow `'inf'`, `'-inf'`, `'nan'` values
+
+Here's an example:
+
+```py
+from pydantic import BaseModel, Field
+class Foo(BaseModel):
+    positive: int = Field(..., gt=0)
+    non_negative: int = Field(..., ge=0)
+    negative: int = Field(..., lt=0)
+    non_positive: int = Field(..., le=0)
+    even: int = Field(..., multiple_of=2)
+    love_for_pydantic: float = Field(..., allow_inf_nan=True)
+foo = Foo(
+    positive=1,
+    non_negative=0,
+    negative=-1,
+    non_positive=0,
+    even=2,
+    love_for_pydantic=float("inf"),
+)
+print(foo)
+#> positive=1 non_negative=0 negative=-1 non_positive=0 even=2 love_for_pydantic=inf
+```
+
+??? info "JSON Schema"
```

Comment:
Ooh I like this. I'll be interested to look at the page when it's done but I think this is a good approach to keeping this page more digestable